apps/builder/app/builder/features/settings-panel/settings-section.tsx

```diff
@@ -50,8 +50,12 @@ export const SettingsSection = () => {
           key={selectedInstance.id}
           placeholder={placeholder}
           value={localValue.value}
-          onChange={(event) => localValue.set(event.target.value.trim())}
-          onBlur={localValue.save}
+          onChange={(event) => localValue.set(event.target.value)}
+          onBlur={(event) =>
+            localValue.finalSave(
```

Comment:
@TrySound maybe take over and see what's the best solution, because we are not going anywhere

Reply:
to me it seemed like the initial version, where he was setting the trimmed value on blur and then calling save was a good call